packages/material/scss/scrollview/_theme.scss

```diff
@@ -0,0 +1,34 @@
+@import "~@progress/kendo-theme-default/scss/scrollview/theme";
+
+@include exports("scrollview/theme/material") {
+    $scrollview-text-opacity: .4 !default;
+    $scrollview-padding-y: 10px !default;
+    $scrollview-padding-x: $padding-y !default;
+
+    .k-scrollview-next,
+    .k-scrollview-prev {
+        span {
+            color: $base-text;
+
+            &:hover {
+
+                &::before {
+                    background: rgba($base-bg, $scrollview-text-opacity);
```

Comment:
When setting the bg color, use the `background-color` property. It may be longer to write, but the `background` shorthand sets a _lot_ of other properties that you probably don't need. This applies to all `background` declarations in the file.